src/pip/_vendor/platformdirs/LICENSE.txt

```diff
@@ -20,4 +20,3 @@ IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
 CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
```

Comment:
Probably mean to remove this file entirely.

Reply:
IANAL but including the licenses is intentional. Distributing the license with the code is a common requirement on OSS licenses.

Reply:
> The above copyright notice and this permission notice shall be included
> in all copies or substantial portions of the Software.

Reply:
Oh. GitHub on mobile was confusing me, it was showing me this file still being named "appdirs.LICENSE.txt".

Yeah totally, for platformdirs agree. So never mind me :)